app/src/components/App/index.tsx

```diff
@@ -72,19 +77,11 @@ class App extends React.Component<AppProps> {
             </Switch>
           </Content>
           <Footer className={styles.footer}>
-            <Menu
-              theme="light"
-              mode="horizontal"
-              style={{ background: '#f0f2f5' }}
-            >
-              <Menu.Item key="1">
-                © 2020{' '}
-                <a href="https://sefglobal.org/">
-                  {' '}
-                  Sustainable Education Foundation - SEF
-                </a>
-              </Menu.Item>
-            </Menu>
+            © 2020{' '}
+            <a href="https://sefglobal.org/">
+              {' '}
```

Comment:
Let's add margin instead of this white space

Reply:
Done